### PR TITLE
fix: add DTEND attribute to events, cf Ticket#6057599

### PR DESCRIPTION
**Pull request by Pierlou**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -21,6 +21,7 @@ def add_event(calendar, name, date):
     event.begin = date.strftime("%Y-%m-%d")
     event.created = datetime.datetime(datetime.date.today().year, 1, 1)
     event.make_all_day()
+    event.end = date.strftime("%Y-%m-%d")
```
Comment (maudetes) [inline main.py:24]: Shouldn't `make_all_day` take care of the end itself?

**Review by maudetes — COMMENTED** [review]
Did you try creating a .ics with this fix and importing it into a client to make sure it still is considered as a an all-day-event and not an event that lasts 24h?

**Comment by Pierlou** [comment]
It does indeed create a 24h event instead of making it an all-day-event. I'll check if there's a way to have both

**Comment by Pierlou** [comment]
After further investigation: the standard for all-day-events is not to have DTEND attributes.
https://stackoverflow.com/questions/1716237/single-day-all-day-appointments-in-ics-files/30249034#30249034
https://www.rfc-editor.org/rfc/rfc5545#section-3.6.1
Our best guess is that the software used by the person who submitted a request on the support doesn't handle these events well.

**Pull request closed without merging**